scripts/generate_refcats_gen3.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# This file is part of ap_verify_hits2015.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Script for ingesting standard refcats that cover this dataset's fields.
+# Running this script allows for updates to the refcats to be incorporated
+# into the dataset.
+
+import argparse
+import logging
+import sys
+
+from esutil import htm
+from astropy.coordinates import SkyCoord
+
+import lsst.log
+from lsst.daf.butler import Butler, CollectionType, DatasetRef, DatasetType, FileDataset
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
+
+
+########################################
+# Fields and catalogs to process
+
+FIELDS = [SkyCoord("10h00m28.800s", "+02d12m36.00s"),  # field 26 (2014:04)
+          SkyCoord("10h20m28.800s", "-06d31m12.00s"),  # field 40 (2014:10)
+          SkyCoord("10h21m52.800s", "-04d57m00.00s"),  # field 42 (2014:09)
+          ]
+FIELD_RADIUS = 2.0  # degrees
+
+# Key is catalog name in source repo, value is name in ap_verify_hits2015.
+REFCATS = {"gaia_dr2_20200414": "gaia",
+           "ps1_pv3_3pi_20170110": "panstarrs",
```

Comment:
It prevents us from having to change configs every time we upgrade the refcats (say, from Gaia DR1 to DR2), and it also keeps us from having configuration bugs that are only detectable when `ap_verify` is run somewhere other than `lsst-devl` (in the early years there were cases where an `ap_verify` dataset was misconfigured, and nobody noticed because the default config picked up the `/datasets/` refcats).

In any case, since this would be a major breaking change to the existing repo, I would like to address it on another ticket, preferably after we remove Gen 2.

Reply:
Ok, lets talk about this on another ticket/at an AP meeting.